README.md

```diff
@@ -9,7 +9,15 @@
 This library is still under construction and needs to be peer reviewed as well as have features added.
 
 ## Usage
-This package contains everything you need to get up and running. If you want to report to a system such as Jaeger or Zipkin you will need to use their NuGet packages.
+This package contains everything you need to get up and running. The meta-package includes the following packages:
+* [Jaeger.Core](https://www.nuget.org/packages/Jaeger.Core/)
+* [Jaeger.Senders.Thrift](https://www.nuget.org/packages/Jaeger.Senders.Thrift/)
+
+### Senders
+The implementation of `Jaeger.Core` is agnostic to any reporting endpoint and could be even used without any in case of logfile reporting. For more information about the sender concept, have a look at the sender [README](src/Jaeger.Core/Senders/README.md).
+
+This is a list of sender implementations known to the Jaeger team:
+* [Jaeger.Senders.Thrift](src/Senders/Jaeger.Senders.Thrift/README.md) (default, included in `Jaeger`)
```

Comment:
L23 - I find it a bit weird that the very first code example is not something that's actually useful. I would prefer to see "the right way" to create a real, prod-quality tracer.

Reply:
It's useful to get the system up and running and making sure that your code is working without having dependencies to other things like environment, senders and so on. That's why I usually like to give simple stand-alone examples and increase the complexity on the way down.

This was also how I started to integrate Jaeger into our application and I found it annoying having to get a minimal example to work first :) Especially since there are lots of ways to get the configuration in on C#, not only environment variables.